src/main/java/entities/Server.java

```diff
@@ -11,6 +11,8 @@
  * a Guild's role list.
  */
 public class Server {
+    public static final String TOPIC_PREFIX = "Topic | ";
```

Comment:
GH won't let me comment but lines 28 and 45 also use this prefix!

Reply:
Good catch! It's fixed in the next set of commits.